templates/favourite/index.php

```diff
@@ -522,6 +525,62 @@ function gtag(){dataLayer.push(arguments)};
 		    <?php } ?>
 
 
+
+        <?php if ($this->countModules('cite-us')) {  ?>
+             <button id="mybtn" type="button" class="favth-btn favth-btn-small" style="position:absolute; top:50%" onClick=getCitation()>Cite Us!</button>
+
+              <!-- Modal -->
+              <div id="myModal" class="modal fade" role="dialog">
```

Comment:
Rename the id as well - Make sure yo don't duplicate ;-)
```suggestion
              <div id="citeus-modal" class="modal fade" role="dialog">
```